sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/Messages.java

```diff
@@ -90,6 +85,11 @@ public enum Messages {
      */
     public static final String EXCEPTION_OCCURRED_WHILE_EMITTING = getMessage("EXCEPTION_OCCURRED_WHILE_EMITTING");
 
+    /**
+     * Encoder is not supported message.
+     */
+    static final String ENCODING_TYPE_NOT_SUPPORTED_ENCODER = getMessage("ENCODING_TYPE_NOT_SUPPORTED_ENCODER");
```

Comment:
Was this change supposed to be there? Seems to be out of place

Reply:
Yes. Otherwise, we're adding to the public surface area.  It's a display message... this class should never have been public.